src/helpers/validation/getFormHasError.ts

```diff
@@ -0,0 +1,7 @@
+export const getFormHasError = ({
+  form,
+  fieldName
+}: {
+  form: { touched: Record<string, any>; errors: Record<string, any> };
```

Comment:
This seems a little complex, should be perhaps declared in it's own interface. 👍